src/test/java/com/browserstack/SeleniumTest.java

```diff
@@ -0,0 +1,24 @@
+package com.browserstack;
+
+import java.net.URL;
+import java.util.HashMap;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumTest {
+    public WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    @SuppressWarnings("unchecked")
+    public void setUp() throws Exception {
+        driver = new ChromeDriver();
```

Comment:
Could you please modify this by instantiating ChromeOptions and passing to ChromeDriver?
( We have one fix undergoing so that user don't have to specify chromeDriver path too.)

Reply:
Done